packages/lwc-language-server/src/lwc-server.ts

```diff
@@ -150,17 +150,19 @@ export default class Server {
                 };
             }
         } else if (await this.context.isLWCJavascript(doc)) {
-            const customTags = this.componentIndexer.customData.map(tag => {
+            if (this.shouldCompleteJavascript(params)) {
+                const customTags = this.componentIndexer.customData.map(tag => {
+                    return {
+                        label: tag.lwcTypingsName,
+                        kind: CompletionItemKind.Folder,
+                    };
+                });
+
                 return {
-                    label: tag.lwcTypingsName,
-                    kind: CompletionItemKind.Folder,
+                    isIncomplete: false,
+                    items: customTags,
                 };
-            });
-
-            return {
-                isIncomplete: false,
-                items: customTags,
-            };
+            }
```

Comment:
so previously we always returned here, but we only return when it passes shouldCompleteJavascript.  Should we move the inner if into the above condition, or should we be returning with no results when no matching tag was found?

Reply:
Ah, I see what you mean. I think an early empty return like what we have for the last else would be a good change here.

Reply:
Added!

Reply:
things that make you go hummmmmmmmm.  How does undefined qualify as a CompletionList? 
weird. 

I did a little digging and think it's b/c `CompletionList` is a namespace and an interface (maybe).  sometimes I wonder if the teams at microsoft talk to each other :).  eh.  def better thanks for the update. 
<img width="1272" alt="Screen Shot 2022-03-15 at 4 53 13 PM" src="https://user-images.githubusercontent.com/76090802/158480060-1032f66c-e855-4b5c-b353-d633a6e82b0b.png">